Sources/Subscription/Managers/SubscriptionManagerV2.swift

```diff
@@ -305,33 +327,27 @@ public final class DefaultSubscriptionManagerV2: SubscriptionManagerV2 {
                 NotificationCenter.default.post(name: .entitlementsDidChange, object: self, userInfo: [UserDefaultsCacheKey.subscriptionEntitlements: newEntitlements])
             }
 
-            if referenceCachedTokenContainer == nil { // new login
+            // Send "accountDidSignIn" notification when login changes
```

Comment:
It is hard to tell without testing it but seems that the order of sending notification should be reversed, in scenario when signing in, the `entitlementsDidChange` notification before `accountDidSignIn` one does not make much sense.